pkg/clean/clean.go

```diff
@@ -58,6 +59,9 @@ func InitializeCleanupService(cf *config.NSXOperatorConfig) (*CleanupService, er
 		NSXConfig: cf,
 	}
 
+	vpcService, vpcErr := vpc.InitializeVPC(commonService)
+	commonctl.ServiceMediator.VPCService = vpcService
+
```

Comment:
just to confirm, only pre-init vpc service in mediator is ok, right? since in mediator there are also other services

Reply:
I’m not sure, at least currently there is no fatal error.